fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/DefaultLoanLifecycleStateMachine.java

```diff
@@ -151,6 +151,11 @@ private LoanStatus getNextState(LoanEvent loanEvent, Loan loan) {
             case LOAN_CREDIT_BALANCE_REFUND:
                 newState = closeObligationsMetTransition();
             break;
+            case LOAN_CHARGEBACK:
+                if (anyOfAllowedWhenComingFrom(from, LoanStatus.CLOSED_OBLIGATIONS_MET, LoanStatus.OVERPAID, LoanStatus.ACTIVE)) {
```

Comment:
Please remove the "LoanStatus.ACTIVE" we dont wanna change from "ACTIVE" state to "ACTIVE" state

Reply:
Done!